src/common/displayFormatters.js

```diff
@@ -0,0 +1,9 @@
+export const formatPhoneNumber = phone => {
```

Comment:
Super basic formatting for the phone number but consistent formatting and validation for all countries is beyond the scope of this ticket